### DISCUSSION TITLE
How to create Order "add record" using api ?

### DISCUSSION THREAD

**Discussion by EvgeniyVorobev** [YetiForceCompany/YetiForceCRM#17327] [How to]
Hello, we need to create order  "add record" button on the screen . Using Rest Api with php language, can you give some instruction. Is thet real ? What method need to be used ?

<img width="1854" alt="image" src="https://github.com/YetiForceCompany/YetiForceCRM/assets/20683521/c362ef30-6f6a-42ed-9309-fd11dcea403d">

**Comment by apcloic** [comment]
https://doc.yetiforce.com/api

**Reply by EvgeniyVorobev** [reply]
<img width="1090" alt="image" src="https://github.com/YetiForceCompany/YetiForceCRM/assets/20683521/520f0d62-2833-4308-85ad-81c6efd338d2">
As I understand it, this method is responsible for this?
But here it is not clear how to pass the parameters; the scheme assigns one description field. In which there are fields field_name_1, field_name_2. What are these fields and what do they do? How can I compare them with those in CRM when creating a record?
